File: neetcode.io/01-arrays/01-ContainsDuplicate/go/main.go
// Copyright (c) 2023 Furukawa, Atsushi <atsushifx@aglabo.com>
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

package main

// modules
import (
	"fmt"
)

// main
func main() {
	var ans bool

	var arr1 []int = []int{1, 2, 3, 1}
	ans = containsDuplicate(arr1)
	fmt.Printf("%v %v\n", arr1, ans)

	var arr2 []int = []int{1, 2, 3, 8}
	ans = containsDuplicate(arr2)
	fmt.Printf("%v %v\n", arr2, ans)
}

File: neetcode.io/01-arrays/01-ContainsDuplicate/ts/main.ts
// Copyright (c) 2023 Furukawa, Atsushi <atsushifx@aglabo.com>
//
// This software is released under the MIT License.
// https://opensource.org/licenses/MIT

// import solver
import solver from './solver.js';

main()
// main
function main() {
	let ans: boolean;

	const arr1: number[] = [1, 2, 3, 1]
	ans = solver.containsDuplicate(arr1)
	console.log(arr1, " answer : " + ans)

	const arr2: number[] = [1, 2, 3, 4]
	ans = solver.containsDuplicate(arr2)
	console.log(arr2, " answer : " + ans)
}

